modules/claims_api/app/controllers/claims_api/v1/claims_controller.rb

```diff
@@ -64,8 +64,19 @@ def fetch_errored(claim)
 
       def format_evss_errors(errors)
         errors.map do |error|
-          formatted = error['key'] ? error['key'].gsub('.', '/') : error['key']
-          { status: 422, detail: "#{error['severity']} #{error['detail'] || error['text']}".squish, source: formatted }
+          if error&.dig('text')
+            formatted = error['key']
```

Comment:
I think this will always get triggered, with the error returned that initially caused this I expected to go the the `elsif`, but it triggered this one because there is a `text` value in the object